### ISSUE TITLE
Debian rpi repo key is now out of date

### ISSUE THREAD

**Issue by JiffB** [signalwire/freeswitch#1579]
Hi folks,

All is in the title, one hour ago, I launched an apt update on my raspberry pi 4 that answered freeswitch bullseye is Unauthorized because:
```
Err:5 https://files.freeswitch.org/repo/deb/debian-release bullseye InRelease       
  401  Unauthorized [IP: 190.102.98.174 443]
Err:6 https://files.freeswitch.org/repo/deb/rpi/debian-release bullseye InRelease   
  401  Unauthorized [IP: 190.102.98.174 443]
Reading package lists... Done                        
E: Failed to fetch http://files.freeswitch.org/repo/deb/debian-release/dists/bullseye/InRelease  401  Unauthorized [IP: 190.102.98.174 443]
E: The repository 'http://files.freeswitch.org/repo/deb/debian-release bullseye InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://files.freeswitch.org/repo/deb/rpi/debian-release/dists/bullseye/InRelease  401  Unauthorized [IP: 190.102.98.174 443]
E: The repository 'http://files.freeswitch.org/repo/deb/rpi/debian-release bullseye InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
Can you generate a new key (and not put it at the same place, as any request is now forbidden) ?

Jean-Yves

**Comment by crienzo** [comment]
Hello - check here  https://freeswitch.org/confluence/display/FREESWITCH/Debian

**Comment by JiffB** [comment]
Ok, I opened an account on got a toekn, it works for sound files, but the rpi repo is now complaining it hasn't the public key to check the repo signature, which doesn't seems available (my add-a-key script contains these public key servers: pgp.mit.edu, keys2.kfwebs.net & fks.pgpkeys.eu) :
```
Hit:5 https://freeswitch.signalwire.com/repo/deb/debian-release bullseye InRelease 
Get:6 https://freeswitch.signalwire.com/repo/deb/rpi/debian-release bullseye InRelease [2,877 B]
Err:6 https://freeswitch.signalwire.com/repo/deb/rpi/debian-release bullseye InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D76EDC7725E010CF
Reading package lists... Done
W: GPG error: https://freeswitch.signalwire.com/repo/deb/rpi/debian-release bullseye InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY D76EDC7725E010CF
E: The repository 'https://freeswitch.signalwire.com/repo/deb/rpi/debian-release bullseye InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
So, where can I found this public key?

Also, for my own information, what is the advantage to force people to open an account @signalwire.com plus get an Id token instead of using a regular repo, like before?
